src/_resources/case-study-cash-app.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2022-11-01
+title: |-
+  Cash App Case Study: Instead of using cash, Monterey & Sacramento riders are choosing to tap the Cash App Card,
+  a free Visa debit card
+asset: Cal-ITP.CashApp.CaseStudy.pdf
```

Comment:
This link is interesting.. actually doesn't link to the full case study, just a one-pager: https://deploy-preview-144--cal-itp-website.netlify.app/assets/Cal-ITP.CashApp.CaseStudy.pdf

Is that the intent? 

Also, this  headline is sentence-case but the rest are title-case.

Reply:
> doesn't link to the full case study, just a one-pager

Yeah we have 2 versions, I'm not sure why / which one is used for what audience. But the [Resources catalog](https://docs.google.com/spreadsheets/d/1xJpgr8JBG9TgZF2mHB8Vjnhu453No4dHoNjFIhb0FQw/edit#gid=0) pointed to this bitly: https://bit.ly/3RWIcEP?r=lp which points to the single-page version.

> Also, this headline is sentence-case but the rest are title-case

Yeah I just copied directly from the Catalog entry?

@mrose914 thoughts on the above questions?

Reply:
Ah ~ Even in the Google Drive, there's only a _draft_ of the full case study. The one pager is fine for this PR.

Reply:
Marking this as resolved for the time being. @mrose914 If we're allowed/want to put the actual case study up, we can do it in a subsequent Issue and Pull Request.